src/Form/ImageTriksType.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\ImageTriks;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class ImageTriksType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('LienImgTriks')
```

Comment:
tu ne vas pas avoir besoin d'un type file pour les upload ??

Reply:
```suggestion
            ->add('LienImgTriks', FileType::class)
```

J'imagine que je dois faire quelque chose comme ça et dans ArticleTriksType ceci : 

```suggestion
           ->add('imageTriks', CollectionType::class, [
                'entry_type' => ImageTriksType::class,
                'allow_add' => true,
                'label' => false,
                'mapped' => false,
            ])
```

Mais en faisant ça je reçois plus un type file dans mon ArticleController mais une instance de ImageTriks ce qui fait que cele ne fonctionne plus : 
                     // On génère un nouveau nom de fichier
                    $fichier = md5(uniqid()) . '.' . $imageFile->guessExtension();
                    // var_dump($fichier);

                    // On copie le fichier dans le dossier uploads
                    $imageTriks->move(
                        $this->getParameter('images_directory'),
                        $fichier
                    );
Vu qu'il me faut un type File pour ça. 
Ou alors j'ai pas compris, ce qui est sûrement le cas